Classes/EventStore/EventTypesFilter.php

```diff
@@ -0,0 +1,60 @@
+<?php
+namespace Neos\Cqrs\EventStore;
+
+/*
+ * This file is part of the Neos.EventStore package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+use TYPO3\Flow\Annotations as Flow;
+
+class EventTypesFilter implements EventStreamFilterInterface
```

Comment:
I'm not really happy with this yet, but at least it makes it quite explicit:
To create a filter based on event types, use this `EventTypesFilter`, to filter for the stream name use the `StreamNameFilter`.. The empty getters are a bit tedious though